components/button/button-icon.js

```diff
@@ -122,6 +122,7 @@ class ButtonIcon extends ThemeMixin(ButtonMixin(VisibleOnAncestorMixin(RtlMixin(
 				.d2l-button-icon {
 					height: 0.9rem;
 					width: 0.9rem;
+					pointer-events: none;
```

Comment:
This is for the weird case where the icon within the button was getting a pointer mouse and preventing click.

Reply:
Is there a way I can reproduce this? Just want to understand better when this is/was coming up.

Reply:
I tried removing this earlier today to reproduce and I wasn't able to. Seems like maybe it was just a coincidence that it worked after adding this. I'm fine to remove it until/unless we can reproduce. It only happened in Polypane.

Reply:
Sounds good.